config/v1/0000_10_config-operator_01_infrastructure.crd.yaml

```
@@ -173,6 +173,18 @@ spec:
                   used by components like the web console to tell users where to find
                   the Kubernetes API.
                 type: string
+              controlPlaneTopology:
+                description: ControlPlaneTopology expresses the expectations for operands
+                  that normally run on control nodes. The default is 'HighlyAvailable',
+                  which represents the behavior operators have in a "normal" cluster.
+                  The 'SingleReplica' mode will be used in single-node deployments
+                  (developer and production) for example, and the operators should
```

Since this is user-facing, we don't need to differentiate between the types of deployments.

```suggestion
                  and the operators should
```

---

changed